docs/release_notes/release_notes-0.14.0.rst

```diff
@@ -19,6 +19,7 @@ Key Features and Updates
   * FIX-#4208: Fix lazy metadata update for `PandasDataFrame.from_labels` (#4209)
 * Performance enhancements
   * FIX-#4138, FIX-#4009: remove redundant sorting in the internal '.mask()' flow (#4140)
+  * FIX-#4183: Stop shallow copies from creating global shared state. (#4184)
```

Comment:
@mvashishtha, why are this release note and the PR description different? These should be aligned.

Reply:
@RehanSD asked me to make the release note describe the bug that this PR fixes. I've now made the PR description the same as the release note.

Reply:
That should've been done before merge as we now have `Make frame _siblings an instance variable.` in the commit history. Please keep in mind this in the future.